### DISCUSSION TITLE
Build failed for iOS release mode by indicating the target version in Podfile

### DISCUSSION THREAD

**Discussion by jsanz1209** [flutter-stripe/flutter_stripe#560] [Q&A]
Hi everyone!

I've integrated this plugin in my flutter project, but to my surprise I can't build on release mode for iOS.

I am facing with problems related to other libraries when I'm building my app:

 /Users/joseignaciosanzgarcia1/monkeybox_app/ios/Runner/GeneratedPluginRegistrant.m:12:9: fatal error: module 'flutter_inappwebview' not found
    @import flutter_inappwebview;

I tried removing the flutter_inappwebview library, but I faced other similar problem again:

 /Users/joseignaciosanzgarcia1/monkeybox_app/ios/Runner/GeneratedPluginRegistrant.m:30:9: fatal error: module 'shared_preferences' not found
    @import shared_preferences;

I have noticed that this has started to happen after setting up the target platform version to 11.0 in the PodFile.

Does anyone know how to resolve this?

Thanks in advance!

**Comment by jsanz1209** [comment]
Why is this moved to closed discussion?

**Reply by jonasbark** [reply]
It's not a closed discussion. It has been moved to Q&A as it's very likely a configuration issue. I'm sure it would help if you can provide a simple project where this behavior is reproducible.
